### ISSUE TITLE
API call CORS issue

### ISSUE THREAD

**Issue by isaacbizteg** [reZach/secure-electron-template#91]
Hi, I'm using axios POST method for API and I get the CORS error. I saw somebody suggest **webSecurity: false**, but I think it defeats the purpose of a secure app. Anyone have a workaround for this? Thanks in advance

**Comment by reZach** [comment]
I think you'll need to provide more code snippets for us to help you with your particular question @isaacbizteg.

**Comment by isaacbizteg** [comment]
Hi @reZach, sorry let me rephrase 😄 I want to implement API call using axios, is there any settings need to be done to allow the call?

**Comment by isaacbizteg** [comment]
Dropped question
